### ISSUE TITLE
mbfan.conf breaks the post install test

### ISSUE THREAD

**Issue by amicus-erez** [dgraziotin/mbpfan#168]
Hi,

https://github.com/dgraziotin/mbpfan/blob/master/mbpfan.conf

Has the min_fan_speed and max_fan_speed variables commented out.
These are copied to /etc/mbfan.conf and result in a failed test, see below.
Cordially, 
 Erez

```
make[1]: Leaving directory '/home/erez/Downloads/mbpfan-2.1.0'
/usr/sbin/mbpfan -f -v -t
Starting the tests..
It is normal for them to take a bit to finish.
Using new sensor path for kernel >= 3.15.0 or some CentOS versions with kernel 3.10.0
Found hwmon path at /sys/devices/platform/coretemp.0/hwmon/hwmon2/temp
Found 5 sensors
Found 2 fans
Using new sensor path for kernel >= 3.15.0 or some CentOS versions with kernel 3.10.0
Found hwmon path at /sys/devices/platform/coretemp.0/hwmon/hwmon2/temp
Found 5 sensors
Could not read min_fan_speed from config file 
Tests run: 4
```

**Comment by gaul** [comment]
Duplicate of #164?

**Comment by amicus-erez** [comment]
Possibly,  I uncommented those lines in source folder and the tests passed.
Those lines were still commented out in the master branch, so I just filed the issue.
I used the latest tarball, not a repo clone.


It is a fantastic project and I am very grateful for it.
Let me know if I could provide more data.
My machine is a mackbook pro 15 early 2013..
 with a busted and disabled nvidia gpu. 
So far so good.

**Comment by gaul** [comment]
Could you test with the latest master?

**Comment by TXMXenon** [comment]
I was getting the same issue and took the advice in #164 and tried the master and now I'm dealing with the same thing as #169 
```
mint@mint-drive:~$ sudo mbpfan -f -t -v
Starting the tests..
It is normal for them to take a bit to finish.
Error: max_fan_speed value is not 6200 
Tests run: 5
mint@mint-drive:~$ sudo mbpfan -f -v -t
Starting the tests..
It is normal for them to take a bit to finish.
Using new sensor path for kernel >= 3.15.0 or some CentOS versions with kernel 3.10.0
Found hwmon path at /sys/devices/platform/coretemp.0/hwmon/hwmon2/temp
Found 5 sensors
Found 2 fans
Using new sensor path for kernel >= 3.15.0 or some CentOS versions with kernel 3.10.0
Found hwmon path at /sys/devices/platform/coretemp.0/hwmon/hwmon2/temp
Found 5 sensors
Error: Invalid Higher temp test (if fan was already spinning high, this is not worrying) 
Tests run: 3
```
My config file (/etc/mbpfan.conf):
```
[general]
# see https://ineed.coffee/3838/a-beginners-tutorial-for-mbpfan-under-ubuntu for the values
# 
# mbpfan will load the max / min speed of from the files produced by the applesmc driver. If these files are not found it w$
# by setting the values for the speeds in this config it will override whatever it finds in:
# /sys/devices/platform/applesmc.768/fan*_min
# /sys/devices/platform/applesmc.768/fan*_max
# or the defaults.
#
# multiple fans can be configured by using the config key of min_fan*_speed and max_fan*_speed
# the number used will correlate to the file number of the fan in the applesmc driver that are used to control the fan spee$
#
min_fan1_speed = 3000   # put the *lowest* value of "cat /sys/devices/platform/applesmc.768/fan*_min"
max_fan1_speed = 6200   # put the *highest* value of "cat /sys/devices/platform/applesmc.768/fan*_max"
low_temp = 61                   # try ranges 55-63, default is 63
high_temp = 66                  # try ranges 58-66, default is 66
max_temp = 84                   # take highest number returned by "cat /sys/devices/platform/coretemp.*/hwmon/hwmon*/temp*_$
polling_interval = 1    # default is 1 seconds
```

**Comment by gaul** [comment]
Please test with the recently released [2.1.1](https://github.com/dgraziotin/mbpfan/releases/tag/v2.1.1) which should address this issue.
